views/js/pciCreator/dev/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -518,13 +526,13 @@ define([
         createToolbar: function createToolbar() {
             var self = this,
                 availableTools = {
-                    frac:   { label: '<sup>x</sup>&frasl;<sub>y</sub>',latex: '\\frac',fn: 'cmd',desc: 'Fraction' },
+                    frac:   { label: 'x/y',         latex: '\\frac',    fn: 'cmd',      desc: 'Fraction' },
                     sqrt:   { label: '&radic;',     latex: '\\sqrt',    fn: 'cmd',      desc: 'Square root' },
                     exp:    { label: 'x&#8319;',    latex: '^',         fn: 'cmd',      desc: 'Exponent' },
                     log:    { label: 'log',         latex: '\\log',     fn: 'cmd',      desc: 'Log' },
                     ln:     { label: 'ln',          latex: '\\ln',      fn: 'cmd',      desc: 'Ln' },
                     e:      { label: '&#8494;',     latex: '\\mathrm{e}',fn: 'write',   desc: 'Euler\'s constant' },
```

Comment:
To be accurate, we don't have the right label for the number e.
We are displaying this one: https://en.wikipedia.org/wiki/Estimated_sign
It would be more correct to change it to a simple `e` (I didn't find any special char for it)